package.json

```diff
@@ -35,15 +35,18 @@
     "lint": "eslint  \"packages/**/*.{js,ts}\" --fix",
     "compile": "cd packages/language-server && npm run build && cd ../vscode && npm run build && cd .. && cd ..",
     "watch": "npm run build -- -w",
-    "vsce:publish": "sh scripts/publish.sh",
-    "vsce:check": "npm run shell:lint && sh scripts/check-update.sh",
-    "vsce:bump": "sh scripts/bump.sh",
-    "shell:lint": "shellcheck -x scripts/**/*.sh",
+    "shell:lint": "shellcheck -x scripts/*.sh",
     "test": "sh scripts/e2e.sh",
     "testScripts": "jest __tests__",
-    "check:lsp": "sh scripts/lsp-version.sh",
-    "setup:patch": "sh scripts/patch/setup.sh",
-    "openvsx:publish": "cd packages/vscode && ovsx publish -p"
+    "openvsx:publish": "cd packages/vscode && ovsx publish -p",
+    "check_for_update": "sh scripts/check_for_update.sh",
+    "bump_prisma_version_in_scripts": "sh scripts/bump_prisma_version.sh",
+    "bump_extension_version_in_scripts": "node scripts/bump_extension_version.js",
+    "setup:branch": "sh scripts/setup_branch.sh",
+    "change_readme": "sh scripts/change_readme.js",
+    "bump_versions_in_repo": "sh scripts/update_versions.sh",
```

Comment:
Looking at the file, this seems to do a lot more than just updating any versions.
Not super sure what it does though.

Reply:
I renamed it to `update_package_json_files`. In the vscode folder, the name, version, description, displayName and preview is modified based on an insider or stable version. In the root folder, the version is modified, and in the language-server folder both the version and the Prisma CLI version is updated.

Reply:
Yes, that makes more sense.